src/Akka.Persistence.Sql/Config/JournalConfig.cs

```diff
@@ -71,6 +73,15 @@ public interface IProviderConfig<TTable>
         string DefaultSerializer { get; }
     }
 
+    public interface IPluginConfig
```

Comment:
For consistency sake, tag separator and tag mode configuration will be standardized to this interface and being stored inside the `IProviderConfig` config class.